scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixFileEvaluation.java

```diff
@@ -18,11 +19,22 @@
 
     /**
      *
-     * @return boolean true if there is no error when scalafix has been evaluated on the file
+     * @return boolean true if there is no error except LintError when scalafix has been evaluated on the file
```

Comment:
Now that `LintError` is no longer something that `getError` can return, I think it's better not to mention it
```suggestion
     * @return boolean true if scalafix was successfully evaluated on the file
```